api-reference/beta/resources/securityprofiles.md

```diff
@@ -0,0 +1,33 @@
+# Security Profiles resource type
+
+Security Profiles represent security properties and an inventory of related entities (hosts, users, IPs, etc) to provide additional context for Alerts and targets for Actions. Security Profiles are populated by integrated solutions at their discretion.

```

Comment:
I haven't seen your metadata, but do I guess correctly this is an entity called "securityProfiles"? If so, please use "securityProfiles resource type" as the topic title.
In general, a resource type topic title should lead with the entity or complex type name as it is defined in the metadata.

Reply:
Why are "Alerts" and "Actions" capitalized? Did you mean certain entity names (in which case they should still be lower camel case), and you wanted to add links to them?

Reply:
@angelgolfer-ms we don't have an entity called securityProfiles, we wanted to relate the 5 SecurityProfile entities under this one file that way they show up under this file in the TOC